tests/fixtures/SwaggerServiceTest/tmp_data_get_user_request_without_request_class.json

```diff
@@ -0,0 +1,99 @@
+{
+    "openapi": "3.1.0",
+    "servers": [
+        {
+            "url": "http:\/\/localhost"
+        }
+    ],
+    "paths": {
+        "/users/{id}/assign-role/{role-id}": {
+            "get": {
+                "tags": [
+                    "users"
+                ],
+                "consumes": [],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "path",
+                        "name": "id",
+                        "description": "",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    {
+                        "in": "path",
+                        "name": "role-id",
+                        "description": "",
+                        "required": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/getUsers{id}assignRole{roleId}200ResponseObject",
+                                    "type": "object"
+                                },
+                                "example": {
+                                    "id": 2,
+                                    "name": "first_client",
+                                    "likes_count": 23,
+                                    "role": {
+                                        "id": 2,
+                                        "name": "client"
+                                    },
+                                    "type": "reader"
+                                }
+                            }
+                        }
+                    }
+                },
+                "security": [],
+                "description": ""
```

Comment:
```diff
                 "security": [],
                 "description": ""
-                "summary" => "test"
-                "deprecated" => false
```
The final change is that these keys were removed because the `saveDescription` was not called in the `parseRequest` method.
https://github.com/RonasIT/laravel-swagger/blob/07aab91443233d767184b7f2d73f60610380e4bc/src/Services/SwaggerService.php#L293